views/forms/DropdownCheckboxGroup/index.jsx

```diff
@@ -23,88 +24,54 @@ const MenuProps = {
 const DropdownCheckboxGroup = ({
   label,
   initialValues,
-  onChange,
-  resetOptions,
-  onReset,
+  name,
+  control,
+  onClear,
+  onClose,
+  onSelectAll,
 }) => {
-  const options = Object.keys(initialValues)
-  const selectedValues = Object.keys(initialValues).filter(
-    (k) => initialValues[k].value === 1
-  )
-  const [selectedValue, setSelectedValue] = useState(selectedValues)
-
-  const handleChange = (event) => {
-    const {
-      target: { value },
-    } = event
-    if (typeof value === 'string') {
-      setSelectedValue(value.split(','))
-      onChange(label, value.split(','))
-    } else {
-      setSelectedValue(value)
-      onChange(label, value)
-    }
-  }
-
-  const handleClearSelection = () => {
-    setSelectedValue([])
-    onChange(label, [])
-  }
-
-  const handleSelectAll = () => {
-    setSelectedValue(options)
-    onChange(label, options)
-  }
-
-  useEffect(() => {
-    if (resetOptions) {
-      const selectedValues = Object.keys(initialValues).filter(
-        (key) => initialValues[key].value === 1
-      )
-      setSelectedValue(selectedValues)
-      onReset()
-    }
-  }, [resetOptions])
+  const { field } = useController({ name, control })
 
   return (
     <Select
       labelId={`multi-chip-label-${label}`}
+      key={`multi-chip-label-${label}`}
       id={`multi-chip-${label}`}
+      data-testid={`multi-chip-${label}`}
       multiple
-      value={selectedValue}
-      onChange={handleChange}
-      input={<OutlinedInput id={`select-multiple-${label}`} label={label} />}
+      {...field}
+      input={
+        <OutlinedInput data-testid={`select-multiple-${label}`} label={label} />
+      }
       renderValue={(selected) => {
         const EtcChip =
           selected.length > 5 ? <Chip key="etc" label="..." /> : <></>
 
-        return (
-          <Box sx={{ display: 'flex', flexWrap: 'nowrap', gap: 0.5 }}>
-            {selected
-              .slice(0, 5)
-              .map((value) => {
-                return <Chip key={value} label={value} />
-              })
-              .concat([EtcChip])}
-          </Box>
-        )
+        return selected
+          .slice(0, 5)
+          .map((value, i) => (
+            <Chip key={`chip-${name}-${label}-${i}`} label={value} />
```

Comment:
```suggestion
            <Chip key={value} label={value} />
```

Reply:
The cohort filters and the recruitment filters have an option with the same name. To simplify it i'll remove this part `-${i}`

Reply:
they have options with the same value? It's fine to have keys with the same value in different maps but I would be really surprised if we had options with the same value for the same filter.

Reply:
They are in different maps

Reply:
[1751eb6](https://github.com/AMP-SCZ/dpdash/pull/738/commits/1751eb64f2b0f1c2e8ad3e3722eab80cc7c314c2)